src/Analyzer.Reports.UnitTests/TestCases.cs

```diff
@@ -70,7 +72,7 @@ public class TestCases
                         HelpUri = "https://domain.com/help",
                         Passed = false,
                         Evaluations = Enumerable.Empty<MockEvaluation>(),
-                        Result = new MockResult { Passed = false, LineNumber = 65 }
```

Comment:
Is it useful to keep the words `LineNumber = `?

Reply:
Could you clarify? This is just updating all results from line number to source location in test cases.

Reply:
I think it was updated or I mis-saw. I don't notice it anymore.